### DISCUSSION TITLE
Add focus option to Applications (menu) groups/layouts

### DISCUSSION THREAD

**Discussion by nerf** [rxhanson/RectanglePro-Community#241] [Ideas]
When using Applications layouts and keybinding is triggered for a group it would be great if focus can move to one of the apps in the group. User should have the option to pick which app get focused.

Edit: typo

**Comment by rxhanson** [comment]
Maybe a checkbox in the app layout settings for "focus first app in list"?

**Reply by nerf** [reply]
Yes, that would be great. Focus first app in the list will be great to have.

**Comment by AziEI** [comment]
Hi!

Is there any plans to add this?

Thank you!
